strapi/strapi
# Make component icons optional

### What does it do?

This PR aims to make the icon, that needs to be selected during creation, optional.

### Why is it needed?

To ease the setup process.

### How to test it?

1. Create a component without an icon
2. Use the component in all places a component can be used (dynamic zone, as single and repeatable component)
3. Validate in the content manager and content-type-builder that a fallback icon is displayed everywhere

### Related issue(s)/PR(s)

- Fixes https://github.com/strapi/strapi/issues/12741
